### PR TITLE
Adding Accordeon + Filters

### PR DESCRIPTION
**Pull request by yannick-ipedis**

### CODE REVIEW

File: Filtres/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    
+    <div class="projects">
+      <div class="project rouge" data-filter="rouge">ROUGE 1</div>
+      <div class="project bleu" data-filter="bleu">BLEU 1</div>
+      <div class="project rouge" data-filter="rouge">ROUGE 2</div>
+      <div class="project hide rouge" data-filter="rouge">ROUGE 3</div>
+      <div class="project bleu" data-filter="bleu">BLEU 2</div>
+      <div class="project rouge" data-filter="rouge">ROUGE 4</div>
+    </div>
+    <script src="scripts.js"></script>
```
Comment (yanis-git) [inline Filtres/index.html:28]: @yannick-ipedis this file seams to be missing on your PR

**Comment by yannick-ipedis** [comment]
Adding filter html file

File: Filtres/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
```
Comment (yanis-git) [inline Filtres/index.html:6]: This file seams to be missing on your PR

File: Filtres/index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (yanis-git) [inline Filtres/index.html:2]: lang is en but content seams to be in french

**Review by yanis-git — CHANGES_REQUESTED** [review]

**Pull request merged**